zebra-state/src/service/non_finalized_state/arbitrary.rs

```diff
@@ -49,19 +54,34 @@ impl Strategy for PreparedChain {
     fn new_tree(&self, runner: &mut TestRunner) -> NewTree<Self> {
         let mut chain = self.chain.lock().unwrap();
         if chain.is_none() {
-            // Only generate blocks from the most recent network upgrade
-            let mut ledger_state = LedgerState::default();
-            ledger_state.network_upgrade_override = None;
+            // Disable NU5 for now
+            // `genesis_strategy(None)` re-enables the default Nu5 override
+            let ledger_strategy = LedgerState::genesis_strategy(Canopy);
```

Comment:
@oxarbitrage this is the NU5 change you'll need to add in your PR after you merge